templates/traefik.yml.j2

```
@@ -79,6 +94,12 @@ certificatesResolvers:
         entrypoint: {{ devture_traefik_config_certificatesResolvers_acme_httpChallenge_entrypoint | to_json }}
 {% endif %}
 
+{% if devture_traefik_config_metrics_prometheus_enabled %}
+metrics:
+  prometheus:
+    entryPoint: metrics
```

```suggestion
    entryPoint: {{ devture_traefik_config_metrics_prometheus_entrypoint | to_json }}
```

I've introduced this new `devture_traefik_config_metrics_prometheus_entrypoint` variable in another review comment.